### ISSUE TITLE
[Crash] IllegalArgumentException: Bound must be positive

### ISSUE THREAD

**Issue by evhub** [SuperMartijn642/Formations#9]
**Version Info**
- Minecraft 1.20.1
- Formations 1.0.2
- Formations Nether 1.0.4
- Formations Overworld 1.0.3

**What mod loader are you using?**: Forge 47.3.0

**Steps to Reproduce**

The server crashed upon generating an RFTools Dimensions dimension.

**Crash report**

[Crash Report](https://gist.github.com/evhub/35ea2710c8f4c7e7d5945f78fa585472)

**Comment by SuperMartijn642** [comment]
Yup, I was generating a random number between the highest and lowest block when trying to find a placement location for a structure, but when the highest and lowest block where the same, it would end up with a range of 0 leading to the crash.
It should be fixed now in Formations 1.0.4.
Thank you for reporting the issue!
